ci_pod/create_ci_pod.sh

```diff
@@ -27,6 +30,7 @@ INTERNAL_URL=""
 
 CI_POD_PATH=""
 
+mkdir -p /tmp/benchmark-runner-run-artifacts
```

Comment:
At some point we need to get away from hard coded pathnames in /tmp and use some mktemp call, but that's just an aside.